### PR TITLE
ScanError: reword error for missing consumed capacity

### PR DESCRIPTION
**Pull request by ikonst**

### CODE REVIEW

File: pynamodb/connection/base.py [hunk 1]
```diff
@@ -1179,8 +1179,8 @@ def rate_limited_scan(self,
                         if allow_rate_limited_scan_without_consumed_capacity:
                             latest_scan_consumed_capacity = 0
                         else:
-                            raise ScanError('Rate limited scan not possible because the server did not send back'
-                                            'consumed capacity information. If you wish scans to complete anyway'
+                            raise ScanError('Rate limited scan not possible because the server did not send back '
+                                            'consumed capacity information. If you wish scans to complete anyway '
                                             'without functioning rate limiting, set '
```
Comment (betamoo) [inline pynamodb/connection/base.py]: how about this line?
```'without functioning rate limiting, set '```

Comment (ikonst) [reply pynamodb/connection/base.py]: It has a space in the end of it.

**Comment by betamoo** [comment]
👍

**Comment by ikonst** [comment]
@betamoo can you give me a GitHub approval?

**Pull request merged**